WORKSPACE

```diff
@@ -3,6 +3,11 @@ workspace(name = "io_bazel_rules_go")
 load("@io_bazel_rules_go//go:def.bzl", "go_rules_dependencies", "go_register_toolchains")
 go_rules_dependencies()
 go_register_toolchains()
+register_toolchains(
```

Comment:
Would it be worthwhile to wrap this in a `proto_register_toolchains` function? Developers should probably have something like this to call.

Reply:
Done